src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/InputSkipIfLastAndSharedWithCase.java

```diff
@@ -0,0 +1,113 @@
+package com.puppycrawl.tools.checkstyle.checks.coding;
+
+
+public class InputSkipIfLastAndSharedWithCase
+{
+    void method(int i) {
+        switch (i) {
+            case 1:
+            default: // No violation with the new option is expected
+                break;
+            case 2:
+                break;
+        }
+
+        switch (i) {
+            case 1:
+            default: // No violation with the new option is expected
+            case 2:
+                break;
+            case 3:
+                break;
+        }
+
+        switch (i) {
+            default: // No violation with the new option is expected
```

Comment:
> u could check it does not change original logic.

I wanted confirmation from romani as he wants specific logic change to check.

> we can put default anywhere as long as it is surrounded by case.

@sagar-shah94 This is not true based on post romani made at https://github.com/checkstyle/checkstyle/issues/4078#issuecomment-289211757 .
````
        switch (mode) {
            default: // violation !!!! with the new option is expected
            case 1:
                return 1;
        }
````

Original check: `default` should be last group in switch
New property: `default` should be last label in group of switch, doesn't matter what group it is in.